parachain/runtime/src/lib.rs

```diff
@@ -356,10 +356,10 @@ parameter_types! {
     pub const LaunchPeriod: BlockNumber = 2 * DAYS;
     pub const VotingPeriod: BlockNumber = 2 * DAYS;
     pub const FastTrackVotingPeriod: BlockNumber = 3 * HOURS;
-    pub MinimumDeposit: Balance = 1000; // TODO: adjust
+    pub MinimumDeposit: Balance = 100 * CENTS;
```

Comment:
IMO using cents does not make this more clear. Especially since it's apparently defined is 1/30000th of a unit

Reply:
This is directly taken from the Polkadot runtime, I'm not exactly sure how this should be determined right now.